docs/2.7.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -180,6 +180,99 @@ Finally, you can ping the participant to see if the system is operational
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.7
+^^^^^^^^^^^^^^^^^^^^^^
+Version 2.7 will slightly extend the database schema used. Therefore, you will have to perform the
+database migration steps.
+Alternatively, you can enable the new "migrate and start" mode of Canton, which will trigger an automatic update of the database schema
+when a new minor version is deployed.
+This mode can be enabled by setting the appropriate storage parameter:
+
+.. code:: bash
+
+    canton.X.Y.storage.parameters.migrate-and-start = yes
+
+In order to benefit from the new security features brought by protocol version 5,
+you will have to :ref:`upgrade the domain accordingly <canton_domain_protocol_version_upgrading>`.
+
+Activation of unsupported features
+""""""""""""""""""""""""""""""""""
+
+In order to activate unsupported features, you now need to explicitly enable dev-version-support on the domain (in addition to the non-standard config flag).
+More information can be found in the :ref:`documentation <how-do-i-enable-unsupported-features>`.
+
+Breaking changes around console commands
+""""""""""""""""""""""""""""""""""""""""
+
+**Key rotation**
+The command keys.secret.rotate_wrapper_key now returns a different error code.
+An INVALID_WRAPPER_KEY_ID error has been replaced by an INVALID_KMS_KEY_ID error.
+
+**Adding sequencer connection**
+The configuration of the Sequencer Client has been updated to accommodate multiple sequencers and their endpoints:
+method `addConnection` has been renamed to `addEndpoints` to better reflect the fact that it modifies an endpoint for the sequencer.
+
+Hence, command to add a new sequencer connection to the mediator would be changed to:
+
+.. code:: bash
+
+    mediator1.sequencer_connection.modifyConnections(
+        _.addEndpoints(SequencerAlias.Default, connection)
+    )
+
+
+Unique Contract Keys Deprecation
+""""""""""""""""""""""""""""""""
+The unique-contract-keys parameters for both participant and domain nodes are now marked as deprecated.
+As of this release, the meaning and default value (true) remain unchanged.
+However, contract key uniqueness will not be available in the next major version, featuring multi-domain connectivity.
+If you are already setting this key to false explicitly (preview), this behavior will be the default one after the configuration key is removed.
+If you don't explicitly set this value to false, you are encouraged to evaluate evolving your existing applications and services to avoid relying on this feature.
```

Comment:
are we printing to the logs a warning/info if it is still true?

Reply:
I don't think so. @stefanobaghino-da do you know if a warning is printed ?

Reply:
[I added it to the deprecated fields](https://github.com/DACH-NY/canton/pull/13207/files#diff-29998a0970c1b9ccd7b6a444b7d0090442702644d090e92b2f13e902054f57b6R84-R89) and that should log a message about this if the field is set explicitly. /cc @thibault-da